### PR TITLE
Privacy blog

### PR DESCRIPTION
**Pull request by LauraDuRy**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/blog/post/How-to-put-privacy-first/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+- **Tags** are snippets of code added to a website to track your activity and collect data. For example, a tag can be used to track how many people visit a particular page on a website or to collect information about what products you are viewing. The most well-known tool for this is [Google Tag Manager](https://www.semrush.com/blog/beginners-guide-to-google-tag-manager/). It allows you to manage all of your website tags in one place.
+- **Pixels** are small, transparent images that are embedded in a website. When you visit a website with a pixel, your browser sends information about your visit to the company that owns the pixel. This information can include the user's IP address, browser type, and the URL of the page you’re visiting. It is a beneficial tool to retarget website visitors with banner or social ads.
+- **Cookies,** first-party are set by the website you’ve visited mainly to improve your experience on that website, whereas different websites, such as Google Analytics set third-party cookies. Third-party cookies are the ones that are often used to track you across platforms, mostly meant for advertising purposes.
```
Comment (Meldiron) [inline src/routes/blog/post/How-to-put-privacy-first/+page.markdoc:20]: Other 2 points sounds like a sentence, but this one reads hard. (beginning)

Comment (LauraDuRy) [reply src/routes/blog/post/How-to-put-privacy-first/+page.markdoc]: updated!

File: src/routes/blog/post/How-to-put-privacy-first/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- Twitter Pixel
+- Facebook Pixel
+- Google Ads Pixel
+- Google Analytics Pixel
+
+Most were used to get website analytics, analyze website clicks, measure ad performance, and collect form submissions.
+
+## The consequences of removing marketing pixels, tags, and cookies
+
+The above mentioned allowed us to analyze user behavior, measure the effectiveness of ad campaigns, retarget ads, and, to some extent, personalize the user experience. There are hardly any growth or marketing teams that don’t rely on them, and as stated in [this article from Osano](https://www.osano.com/articles/marketer-friendly-privacy-software), privacy regulations make it a lot harder for growth and marketing teams to do their work. Go to any privacy footer on any website, and you will read an essay on all the data that is being tracked. But this is precisely what we are challenging: does removing pixels, cookies, and tags make growth’s life harder? Let’s take a look.
```
Comment (Meldiron) [inline src/routes/blog/post/How-to-put-privacy-first/+page.markdoc:39]: `this article from Osano` to just `article from Osano`.

Clickable texts should make sense out of context as well. `this` is giving it context.

If you really want to keep `this`, we can do it before link. But I don't think its nessessary.

File: src/routes/blog/post/How-to-put-privacy-first/+page.markdoc [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+- **Open source analytics alternatives.** We use privacy-friendly alternatives like [Plausible](https://plausible.io/about) to help us analyze our website and Console data. It is an open-source analytics platform with one of the highest [privacy standards](https://plausible.io/privacy) I have seen. It helps us get demographic and user behavior data in a privacy-friendly manner.
+- **First-party data.** We get a lot from users, like emails and names from authentication. We use it for login purposes, so you don’t have to enter your login details over and over again and add them to our CRM, HubSpot, for email communications. But with this data also comes great responsibility, and with our recent GDPR compliance announcement, we are not only upholding strict data privacy but also certified with the European General Data Protection Regulation (GDPR) standards.
```
Comment (Meldiron) [inline src/routes/blog/post/How-to-put-privacy-first/+page.markdoc:63]: Doesnt sound like proper beginning of sentence

Comment (Meldiron) [reply src/routes/blog/post/How-to-put-privacy-first/+page.markdoc:63]: `but also certified with the European General ...`
`but are also certified with the ...`

I think adding `are` is grammatically correct. Please check with someone from US/UK, or Grammarly.

**Review by TorstenDittmann — APPROVED** [review]
technical part ✅

File: src/routes/blog/post/How-to-put-privacy-first/+page.markdoc [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+Overall, it made much sense to go down this route and focus on delivering a better experience and enhancing trust with the community. And I think it’s fair to say that with the removal of marketing pixels, cookies, and tags, we didn’t lose anything; we merely gained.
+
+## Building a privacy-friendly and open-source growth stack
+
+We still have the necessary tools to improve our product, the developer experience, and analyze our website traffic. That said, we will continue to explore the possibilities of moving towards a fully open-source and privacy-friendly growth tech stack. Not only do we feel it is the right thing to do, but it is also rooted in our culture. We want to treat you as we want to be treated by any vendors we use. For now, this has been the most sensible decision we could have made for Appwrite and the community. We are excited to take privacy further and further, so who knows where we will stand a year from now.
```
Comment (Meldiron) [inline src/routes/blog/post/How-to-put-privacy-first/+page.markdoc:84]: `We want to treat you as we want to be treated by any vendors we use`

Powerful sentence, I think it should be used in the beginning of the article. It makes it very clear we understand the pain point, and have been on receiving end too.

File: static/images/blog/privacy.png
Comment (christyjacob4) [inline static/images/blog/privacy.png:1]: Can we compress this image or get a smaller sized version of it? 3.19MB might be overkill and lead to slower loading times @LauraDuRy

Comment (LauraDuRy) [reply static/images/blog/privacy.png:1]: on it

**Review by eldadfux — CHANGES_REQUESTED** [review]
Left some comments on Discord.

**Pull request merged**